spec/support/shared_examples/mappings_batch.rb

```diff
@@ -1,3 +1,34 @@
+shared_examples 'creates redirect mapping' do
+  context 'some context' do
```

Comment:
Any reason we're keeping a couple of contexts with slightly vague names?

(apologies if I shouldn't be looking at this, I should really have unsubscribed ;))